src/components/editMeasure/populationCriteria/QDMReporting/QDMReporting.tsx

```diff
@@ -144,47 +155,79 @@ const QDMReporting = () => {
+                  {...formik.getFieldProps("rateAggregation")}
+                  aria-labelledby="rate-aggregation-label"
+                  type="text"
+                  disabled={!canEdit}
+                  name="rate-aggregation"
+                  id="rate-aggregation"
+                  autoComplete="rate-aggregation"
+                  placeholder="Rate Aggregation"
+                  data-testid="rateAggregationText"
+                  {...formik.getFieldProps("rateAggregation")}
+                />
+              </FieldSeparator>
+            </div>
+            <div />
+            <div>
+              <Select
+                name="Improvement Notation"
+                placeHolder={{ name: "Select Improvement Notation", value: "" }}
+                label="Improvement Notation"
+                id="improvement-notation-select"
+                inputProps={{
+                  "data-testid": "improvement-notation-input",
+                }}
                 disabled={!canEdit}
-                name="rate-aggregation"
-                id="rate-aggregation"
-                autoComplete="rate-aggregation"
-                placeholder="Rate Aggregation"
-                data-testid="rateAggregationText"
-                {...formik.getFieldProps("rateAggregation")}
+                data-testid="improvement-notation-select"
+                {...formik.getFieldProps("improvementNotation")}
+                size="small"
+                options={Object.values(improvementNotationOptions).map(
+                  (opt) => (
+                    <MuiMenuItem key={opt.label} value={opt.value}>
+                      {opt.label}
+                    </MuiMenuItem>
+                  )
+                )}
               />
-            </FieldSeparator>
-          </div>
-          <div className="right">
-            <Select
-              name="Improvement Notation"
-              placeHolder={{ name: "Select Improvement Notation", value: "" }}
-              label="Improvement Notation"
-              id="improvement-notation-select"
-              inputProps={{
-                "data-testid": "improvement-notation-input",
-              }}
-              disabled={!canEdit}
-              data-testid="improvement-notation-select"
-              {...formik.getFieldProps("improvementNotation")}
-              size="small"
-              options={Object.values(improvementNotationOptions).map((opt) => (
-                <MuiMenuItem key={opt.label} value={opt.value}>
-                  {opt.label}
-                </MuiMenuItem>
-              ))}
-            />
-          </div>
+            </div>
+
+            {formik.getFieldProps("improvementNotation").value == "Other" && (
+              <div>
+                <FieldLabel htmlFor="imr" id="improvement-notation-other-label">
+                  Improvement Notation (Other)
+                </FieldLabel>
+                <FieldSeparator>
+                  <FieldInput
```

Comment:
I think we should be using input component from design systems.

Reply:
Will address in a different ticket